modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala

```diff
@@ -58,7 +59,10 @@ final case class DagRoutes[F[_]: Async](dagService: DAGService[F], mkDagCell: L0
         .map(L0CellInput.HandleDAGL1(_))
         .map(mkDagCell)
         .flatMap(_.run())
-        .flatMap(_ => Ok())
+        .flatMap {
+          case Right(_)             => Ok()
+          case Left(CellError(msg)) => BadRequest(msg)
```

Comment:
This should be somehow handled on caller-site

Reply:
It is just logged on caller-site. There is used `successful` method from `http4s.Client` which should return false for `BadRequest` status and then warning message should be logged.